### ISSUE TITLE
[Bug] v1.1.0 Login not possible, button unresponsive

### ISSUE THREAD

**Issue by headcase0570** [lencx/ChatGPT#1201]
### Non-ChatGPT bug

- [X] This issue does not occur on chat.openai.com and only occurs on this app.

### Version

1.1.0

### Bug description

Upon starting the ChatGPT app, I am facing two buttons; 'Log in' and 'Sign up'. Neither button is responsive. I have managed to use the settings menu to clear the ChatGPT config, after which the app closes itself. I have cleared the settings several times.
I have a ChatGPT Plus subscription I would like to log in with.

I can click either buttons as much as I want, but nothing happens. I've had this same issue happen on my Windows 11 PC, but there the issue was resolved by clearing the configuration.

### OS

MacOS Big Sur v11.7.10

**Comment by Spidermian** [comment]
Neither button is responsive on OS X Mojave 10.14.6

**Comment by dmatora** [comment]
same issue

**Comment by zwm88** [comment]
After clicking either button, the window becomes a blank page.

**Comment by nabeel-tahir-123** [comment]
Yup issue still exits. Chatgpt site works fine. so no DNS issue. maybe they revoked app connecting ? @lencx  . check asap. login issue is kinda critical

**Comment by jgimeno** [comment]
Same here, Mac OS

**Comment by DonRichards** [comment]
Same here. I've been using this for a while. I just uninstalled it and cleared all configs and then tried installing with brew. I also tried installing the DMG file and got the same results.
## Key Symptoms :
1. I **can** log-in but the UI was very slow to respond
1. I **can** resize the window
1. I can't move the window
1. I can't click on anything
1. I have tried manually switching to another DNS provider (not that I expect that to change an app's ability to be clickable). 

## Specs:
MacOS 14.4.1
ChatGPT Version 1.1.0 (20230803.134227)
